src/dataframerow/dataframerow.jl

```diff
@@ -7,6 +7,12 @@ A `DataFrameRow` is constructed with `view` or `getindex` when one row and a
 selection of columns are requested, or when iterating the result
 of the call to the [`eachrow`](@ref) function.
 
+You can also call a `DataFrameRow` constructor directly using the following syntax:
```

Comment:
```suggestion
The `DataFrameRow` constructor can also be called directly:
```
Above, maybe say "is returned" instead of constructed?

Reply:
fixed